### PR TITLE
Add test launcher and sample tests

### PR DESCRIPTION
**Pull request by lask**

### CODE REVIEW

**Review by floitsch — APPROVED** [review]
LGTM.

File: vscode/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (jstroem) [inline vscode/.vscode/launch.json:4]: comments in json is by default not valid

File: vscode/.vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
```
Comment (jstroem) [inline vscode/.vscode/tasks.json:2]: comments in json is by default not valid

File: vscode/.vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "args": [
+        "--extensionDevelopmentPath=${workspaceFolder}"
+      ],
+      "name": "Launch Extension",
+      "outFiles": [
+        "${workspaceFolder}/out/**/*.js"
+      ],
+      "preLaunchTask": "npm: compile",
```
Comment (jstroem) [inline vscode/.vscode/launch.json:27]: should this be  `npm: test-compile`?

**Review by jstroem — APPROVED** [review]

File: vscode/.vscode/launch.json (hunk 3 above)
Comment (lask) [reply vscode/.vscode/launch.json:24]: Could be. I think, the difference is whether we use the typescript compiler or webpack.

**Pull request merged**